impeller/tessellator/tessellator_unittests.cc

```diff
@@ -78,30 +78,6 @@ TEST(TessellatorTest, TessellatorBuilderReturnsCorrectResultStatus) {
 
     ASSERT_EQ(result, Tessellator::Result::kInputError);
   }
-
-  // More than uint16 points, odd fill mode.
-  {
```

Comment:
This is testing a case that no longer matters since we don't use the tessellator. i suspect it broke because we're now generating fewer line segments, since our cubic division isn't as cranky.

As a follow up I plan to delete most of this code.